_posts/2024-02-06-opa-arrived.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title:  "Open Policy Agent for Trino arrived"
+author: Manfred Moser
+image: /assets/images/logos/opa-small.png
+excerpt_separator: <!--more-->
+show_pagenav: true
+---
+
+Trino now ships with an access control integration using the popular and widely
+used [Open Policy Agent (OPA)](https://www.openpolicyagent.org/) from the Cloud Native
+Computing Foundation. The release of [Trino
+438]({{site.url}}/docs/current/release/release-438.html) marks an important
+milestone of the effort towards this integration.
+
+<!--more-->
+
+## Collaboration and history
+
+Open Policy Agent was first released in 2016 and has gained more and more
+popularity in the ecosystem of cloud native applications and beyond.
+
+Initial efforts for an integration with Trino started at Bloomberg, Stackable,
+Raft, and other places separately and sometimes in parallel, with only partial
+collaboration. You might have first heard about it in August 2022 in the [Trino
+Community Broadcast episode 39]({{site.url}}/episodes/39.html) with a team from
+Raft as guests.
+
+Usage and experience with OPA grew. In the end, Pablo Arteaga from
+[Bloomberg](https://www.techatbloomberg.com/) and Sebastian Bernauer and Sönke
+Liebau from [Stackable](https://stackable.tech/) had the initiative to start a
+pull request to Trino. Their persistence and collaboration led them through many
+review comments, update commits, and even a second PR, to submit a talk and
+eventually present at Trino Summit 2023 about the Open Policy Agent access
+control with Trino and their motivation to move from Apache Ranger to OPA.
+
+## OPA at Trino Summit 2023
+
+The presentation from Pablo and Sönke titled "Trino OPA authorizer - An open
```

Comment:
You should summarize what they talked about in 2-3 sentences

Reply:
Done .. also for the other one